### PR TITLE
fix: tutorial show MOTD and ask for user's email popup

### PR DESCRIPTION
**Pull request by pbosio**

### CODE REVIEW

File: unity-client/Assets/Tutorial/Scripts/TutorialSteps/TutorialStep_ChatAndAvatarExpressions.cs [hunk 1]
```diff
@@ -44,6 +69,45 @@ public override IEnumerator OnStepExecute()
 
             yield return dailyRewardTooltip.ShowAndHideRoutine();
             yield return WaitIdleTime();
+
+            bool hasWelcomeHudPop = false;
+            if (HUDController.i != null && HUDController.i.welcomeHudController != null)
+            {
+                isWelcomeHudVisible = true;
+                hasWelcomeHudPop = true;
+                Utils.UnlockCursor();
```
Comment (BrianAmadori) [inline unity-client/Assets/Tutorial/Scripts/TutorialSteps/TutorialStep_ChatAndAvatarExpressions.cs]: This is not necessary, should be already handled by the welcome popup.

**Review by pravusjif — APPROVED** [review]
I got through all the tutorial and then got the MOTD and the email prompt, great.

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WelcomeHUD/WelcomeHUDController.cs [hunk 2]
```diff
@@ -10,19 +10,18 @@ public class Model : HUDConfiguration
         public bool hasWallet; //TODO(Brian): Use WEB3 param in userProfile
```
Comment (BrianAmadori) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WelcomeHUD/WelcomeHUDController.cs:10]: I implemented this before the profile change. Can we use the profile web3 flag here and remove this? We should remove it from kernel too.

File: unity-client/Assets/Tutorial/Scripts/TutorialSteps/TutorialStep_ChatAndAvatarExpressions.cs (hunk 1 above)
Comment (pbosio) [reply unity-client/Assets/Tutorial/Scripts/TutorialSteps/TutorialStep_ChatAndAvatarExpressions.cs]: Added lock and unlock cursor superpowers to WelcomeHUDController

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WelcomeHUD/WelcomeHUDController.cs (hunk 2 above)
Comment (pbosio) [reply unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WelcomeHUD/WelcomeHUDController.cs:10]: unity receive the profile after WelcomeHUDController is initialized, so we currently can use that flag.
maybe we should address this in a future PR

**Review by BrianAmadori — APPROVED** [review]

**Pull request merged**